From a8fd44cbd283cb9825f9d799da14234148e5a4dd Mon Sep 17 00:00:00 2001
From: Bagas Sanjaya <bagasdotme@gmail.com>
Date: Sun, 19 Mar 2023 14:49:03 +0700
Subject: Documentation: leds: MT6370: Use bullet lists for timing variables

The timing description contains list of timing pattern variables, but it
uses code block without indentation instead. Switch to bullet list as it
is better fit for this purpose.

While at it, substitute "load" for "duty" because the variables control
timing for current load into the device.

Link: https://lore.kernel.org/oe-kbuild-all/202303182310.tB1mUzU7-lkp@intel.com/
Fixes: 4ba9df04b7ac66 ("docs: leds: Add MT6370 RGB LED pattern document")
Reported-by: kernel test robot <lkp@intel.com>
Signed-off-by: Bagas Sanjaya <bagasdotme@gmail.com>
Signed-off-by: Lee Jones <lee@kernel.org>
Link: https://lore.kernel.org/r/20230319074903.13075-4-bagasdotme@gmail.com
---
 Documentation/leds/leds-mt6370-rgb.rst | 18 +++++++++---------
 1 file changed, 9 insertions(+), 9 deletions(-)

diff --git a/Documentation/leds/leds-mt6370-rgb.rst b/Documentation/leds/leds-mt6370-rgb.rst
index ea782797a06df..152a2e5921724 100644
--- a/Documentation/leds/leds-mt6370-rgb.rst
+++ b/Documentation/leds/leds-mt6370-rgb.rst
@@ -45,17 +45,17 @@ Pattern diagram::
           +----------------------------------============------------> Time
           < Tr1><Tr2><   Ton    ><Tf1><Tf2 ><  Toff    >< Tr1><Tr2>
 
-Timing description::
+Timing description:
 
-Tr1:    First rising time for duty 0 to 30%.
-Tr2:    Second rising time for duty 31% to 100%.
-Ton:    On time for duty 100%.
-Tf1:    First falling time for duty 100% to 31%.
-Tf2:    Second falling time for duty 30% to 0%.
-Toff:   Off time for duty 0%.
+  * Tr1:    First rising time for 0% - 30% load.
+  * Tr2:    Second rising time for 31% - 100% load.
+  * Ton:    On time for 100% load.
+  * Tf1:    First falling time for 100% - 31% load.
+  * Tf2:    Second falling time for 30% to 0% load.
+  * Toff:   Off time for 0% load.
 
-Tr1/Tr2/Tf1/Tf2/Ton: 125ms to 3125ms, 200ms per step.
-Toff: 250ms to 6250ms, 400ms per step.
+  * Tr1/Tr2/Tf1/Tf2/Ton: 125ms to 3125ms, 200ms per step.
+  * Toff: 250ms to 6250ms, 400ms per step.
 
 Pattern example::
 
-- 
cgit 

